main/raspberrypi/APKBUILD

```diff
@@ -44,6 +44,7 @@ package() {
 	# nuke the unwanted stuff
 	rm -rf "$pkgdir"/opt/vc/src
 	rm -rf "$pkgdir"/opt/vc/lib/*.a
+	cp -r "$pkgdir"/opt/vc/etc "$pkgdir"
```

Comment:
I don't understand why the etc config is copied. Does it fix something? Should it then be moved instead?

Reply:
This is the reason https://github.com/raspberrypi/userland/commit/e5803f2c986cbf8c919c60278b3231dcdf4271a6

Reply:
Seems like moving it would be the right way.
I'll make a change